### PR TITLE
Add tests for FilePackage.Headers and more todos

### PR DESCRIPTION
**Pull request by victornoel**
This is for #75. See discussion there as to why the todo on `FilePackage` was added.

### CODE REVIEW

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/rpm-adapter/issues/152#issuecomment-620487072))

@g4s8 This is what I know about this job in CT2E6TK9B, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#152](https://github.com/artipie/rpm-adapter/issues/152) is [in scope](http://www.zerocracy.com/policy.html#14) for 1hr
  * The role is `REV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @victornoel[/z](https://www.0crat.com/u/victornoel): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Frpm-adapter%23152%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
This pull request [#152](https://github.com/artipie/rpm-adapter/issues/152) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/225e2ee5-f2f2-49f2-973f-47a7f06d98df); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/rpm/pkg/FilePackageHeadersTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ *  of MetaHeader with two test method: one for the tag being
+ *  present and one for when it is absent (and default value is
+ *  returned).
+ */
+public final class FilePackageHeadersTest {
+    @Test
+    public void parseStringHeader(@TempDir final Path unused) {
+        final Header.HeaderTag tag = Header.HeaderTag.NAME;
+        final String expected = "string value";
+        final Header headers = new Header();
```
Comment (Vatavuk) [inline src/test/java/com/artipie/rpm/pkg/FilePackageHeadersTest.java:49]: @victornoel `header` instead of `headers`

**Comment by victornoel** [comment]
@Vatavuk thank you, it is done

**Comment by 0crat** [comment]
Job was finished in 24 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**